### PR TITLE
@dzucconi - adds follow artists schema and context for following artists module

### PR DESCRIPTION
**Pull request by broskoski**
Comments inline

### CODE REVIEW

File: lib/loaders/total.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { toKey } from '../helpers';
 import gravity from '../apis/gravity';
 import httpLoader from './http';
 
-export const total = key => {
-  return gravity(key)
+export const total = (key, accessToken) => {
```
Comment (broskoski) [inline lib/loaders/total.js]: Optionally pass `accessToken` for authenticated counts

File: schema/home/fetch.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { sortBy, first, forEach, clone } from 'lodash';
 import blacklist from '../../lib/artist_blacklist';
 
 export const featuredFair = () => {
-  return gravity('fairs', { size: 5, active: true }).then((fairs) => {
+  return gravity('fairs', { size: 5, status: 'running', has_full_feature: true }).then((fairs) => {
```
Comment (broskoski) [inline schema/home/fetch.js]: Snuck this in to get actual running fairs.

File: schema/me/follow_artists.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  GraphQLObjectType,
+} from 'graphql';
+
+const FollowArtists = new GraphQLObjectType({
+  name: 'FollowArtists',
+  fields: {
+    artists: {
+      type: new GraphQLList(Artist.type),
+      resolve: (data) => {
+        const artists = data.artists ? data.artists : data;
```
Comment (broskoski) [inline schema/me/follow_artists.js]: This is a little weird, I'm sure theres a better way to do it. For the purposes of the module context, we resolve with an object that has a `context_type` but the normal response is just an array. We want this to resolve with artists in both cases.

Comment (dzucconi) [inline schema/me/follow_artists.js]: This is fine—but:

File: schema/me/follow_artists.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        return map(artists, 'artist');
+      },
+    },
+    total_count: {
+      type: GraphQLInt,
```
Comment (dzucconi) [inline schema/me/follow_artists.js]: I'd prefer this to be nested schema called counts, with a numeral field for the specific set of counts, which would make it consistent with all the other schemas. But you're the consumer.

File: schema/home/context.js [hunk 5]
```diff
@@ -32,14 +33,24 @@ export const HomePageModuleContextTrendingType = create(Trending.type, {
...
 export const moduleContext = {
   iconic_artists: () => {
     return iconicArtists().then((trending) => {
       return assign({}, trending, { context_type: 'Trending' });
     });
   },
   active_bids: () => false,
-  followed_artists: () => false,
+  followed_artists: ({ accessToken }) => {
+    return gravity.with(accessToken)('me/follow/artists', { size: 9, page: 1, total_count: true })
```
Comment (dzucconi) [inline schema/home/context.js]: Don't need to do the total count here.

**Comment by dzucconi** [comment]
Looks good—I made some minor comments. Merge when you're happy though.

File: schema/home/context.js (hunk 5 above)
Comment (broskoski) [inline schema/home/context.js]: 👍

**Pull request merged**